src/types/mailchimp-marketing.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module "@mailchimp/mailchimp_marketing" {
```

Comment:
There is a published type file for this module, but it has the wrong type for `updateListMemberTags`

Reply:
Is there an upstream where you could open a PR, if that's fast?

Reply:
There is, but it might require investigating several other endpoints to do properly, because I suspect there are others with the same problem.